include/cc_helpers.h

```diff
@@ -28,6 +12,22 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+#include <sys/socket.h>
```

Comment:
This is just a 'relocation' diff, only to ensure that the copyright notice consistently appears at the top of each file, wherever it appears.